cluster/log-dump.sh

```diff
@@ -132,7 +132,7 @@ function save-logs() {
     if log-dump-ssh "${node_name}" "sudo systemctl status kubelet.service" &> /dev/null; then
         log-dump-ssh "${node_name}" "sudo journalctl --output=cat -u kubelet.service" > "${dir}/kubelet.log" || true
         log-dump-ssh "${node_name}" "sudo journalctl --output=cat -u docker.service" > "${dir}/docker.log" || true
-        log-dump-ssh "${node_name}" "sudo journalctl --output=cat -k" > "${dir}/kern.log" || true
+        log-dump-ssh "${node_name}" "sudo journalctl --output=short-precise -k" > "${dir}/kern.log" || true
```

Comment:
It seems that we just need to remove `--output=cat`.

`short-precise: The default format with microsecond precision`
https://www.digitalocean.com/community/tutorials/how-to-use-journalctl-to-view-and-manipulate-systemd-logs

But I'm also fine with explicitly add it, up to you. :)